gigaumbrella/values.yaml

```diff
@@ -561,18 +561,26 @@ jenkins:
                           privateKeySource:
                             directEntry:
                               privateKey: "${git-credentials-id-secret-idrsa}"
+                      - basicSSHUserPrivateKey:
+                          scope: GLOBAL
+                          id: xap-builder-ssh
+                          username: xap-builder
+                          description: "Private key access to xap-premium-GitHub"
+                          privateKeySource:
+                            directEntry:
+                              privateKey: "${xap-builder-ssh-secret-idrsa}"
                       - usernamePassword:
                           scope: GLOBAL
                           id: "aws_ses"
                           username: "AKIAYK72STYCODIHYDMO"
                           password: "${smtp-secret-password}"
                           description: "mail_sender"
                       - usernamePassword:
-                          description: "gigaops12-dockerhub"
+                          description: "xapbuilder-dockerhub"
                           id: "docker-hub-cred"
                           password: "${docker-hub-cred-password}"
                           scope: GLOBAL
-                          username: "gigaops12"
+                          username: "xapbuilder"
                       - usernamePassword:
                           description: "git_token"
```

Comment:
Should the git_token be changed as well? Current git_token is for gigaops12.

Reply:
Yes, it should. I checked it vs. the running config and Yuval removed the user from the org anyway.